classifiers/reference_quality/tiktoken_length_classifier/code_snippet_common.md

```diff
@@ -0,0 +1,30 @@
+```python
+import tiktoken
+
+def tiktoken_length_classifier(text: str, encoding_name: str = "cl100k_base") -> str:
+    """
+    @param text: text you want to classify
+    @return: either 'short', 'medium' or 'long' depending on the amount of tokens in a text
+    """
```

Comment:
missing documentation of parameter `encoding_name`

Reply:
Yes true, I have added it.